src/components/cylc/table/Table.vue

```diff
@@ -239,9 +304,73 @@ export default {
           }
           return true
         })
+      this.sortBy.forEach(sortByProperty => {
+        filteredSortedTasks = filteredSortedTasks.sort((taskA, taskB) => {
+          let valueA
+          let valueB
+          const sortDesc = this.sortDesc[this.sortBy.indexOf(sortByProperty)]
+          switch (sortByProperty) {
+          case 'Task':
+            valueA = typeof taskA.node.name !== 'undefined' ? taskA.node.name : ''
+            valueB = typeof taskB.node.name !== 'undefined' ? taskB.node.name : ''
+            return sortDesc ? valueB.localeCompare(valueA) : valueA.localeCompare(valueB)
+          case 'Cycle Point':
+            valueA = taskA.node.cyclePoint !== '' && typeof taskA.node.cyclePoint !== 'undefined' ? (new Date(taskA.node.cyclePoint)).getTime() : 0
+            valueB = taskB.node.cyclePoint !== '' && typeof taskB.node.cyclePoint !== 'undefined' ? (new Date(taskB.node.cyclePoint)).getTime() : 0
+            return sortDesc ? valueB - valueA : valueA - valueB
```

Comment:
Date-time cycle points are ISO8601 format, e.g. `20240502T00+1200` - can `Date().getTime()` cope with that?

Reply:
Alphanumeric sort is probably fine, because the format is the same for all tasks in a given workflow.

Reply:
Just checked, and cyclepoint sorting seems not to be working for a datetime cycling workflow.